OPAL/tac/src/main/scala/org/opalj/tac/cg/android/AndroidEntryPointsFinder.scala

```diff
@@ -0,0 +1,71 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package tac
+package cg
+package android
+
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+import scala.jdk.CollectionConverters._
+import scala.util.control.Exception.allCatch
+
+import org.opalj.br.analyses.SomeProject
+import org.opalj.br.analyses.cg.EntryPointFinder
+import org.opalj.br.Method
+import org.opalj.br.MethodDescriptor
+import org.opalj.br.ObjectType
+
+/**
+ * The AndroidEntryPointFinder considers specific methods of app components as entry points.
+ * It does not work for androidx
+ *
+ * @author Tom Nikisch
+ */
+object AndroidEntryPointsFinder extends EntryPointFinder {
+
+    override def collectEntryPoints(project: SomeProject): Iterable[Method] = {
+        val entryPoints = ArrayBuffer.empty[Method]
+        val defaultEntryPoints = mutable.Map.empty[String, List[(String, Option[MethodDescriptor])]]
+        val defaultEntry = project.config.getConfig("org.opalj.tac.cg.android.AndroidEntryPointsFinder")
+        defaultEntry.root().entrySet().forEach { entry =>
```

Comment:
This looks weird, I have not seen code use the config like this before in OPAL. See e.g. ConfiguredNativeMethodsCallGraphAnalysis for how to do it properly.